### PR TITLE
[DOCS-5072] Add Google Cloud Configuration Guide for Cloud SIEM

### PR DESCRIPTION
**Pull request by maycmlee**
### What does this PR do?

Adds new Cloud SIEM guide.

### Motivation

[DOCS-5072](https://datadoghq.atlassian.net/browse/DOCS-5072)

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

[DOCS-5072]: https://datadoghq.atlassian.net/browse/DOCS-5072?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: content/en/security/cloud_siem/guide/google-cloud-config-guide-for-cloud-siem.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+1. Navigate to Pub/Sub > [Subscriptions][8].
+1. Click **Create Subscription**.
+1. Enter a subscription name.
+1. Select the topic you created earlier.
+1. In **Delivery type**, select **Push**.
+1. Enter the following endpoint URL, replacing `DATADOG_API_KEY` with an existing or new Datadog API key: `https://gcp-intake.logs.{{< region-param key="dd_site" code="true" >}}/api/v2/logs?dd-api-key=<DATADOG_API_KEY>&dd-protocol=gcp`
+1. Configure any additional options for your use case.
+1. Click **Create**.
+
+The Pub/Sub is ready to receive logs from Google Cloud Logging and forward them to Datadog.
```
Comment (cswatt) [inline content/en/security/cloud_siem/guide/google-cloud-config-guide-for-cloud-siem.md:65]:
```suggestion
The pub/sub is ready to receive logs from Google Cloud Logging and forward them to Datadog.
```

File: content/en/security/cloud_siem/guide/google-cloud-config-guide-for-cloud-siem.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+1. Enter the query to filter to the logs you want to export.
+1. Select **Log Router** in the left side menu.
+1. Click **Create Sink**.
+1. Enter a name for the sink.
+1. Click **Next**.
+1. In the **Select Sink Service** dropdown menu, select **Cloud Pub/Sub topic**. 
+1. In the **Select a Cloud Pub/Sub topic**, select the Pub/Sub created earlier.
+1. Click **Create Sink**.
+
+**Note**: It is possible to create several exports from Google Cloud Logging to the same Pub/Sub with different sinks.
```
Comment (cswatt) [inline content/en/security/cloud_siem/guide/google-cloud-config-guide-for-cloud-siem.md:79]:
```suggestion
**Note**: It is possible to create several exports from Google Cloud Logging to the same pub/sub with different sinks.
```

File: content/en/security/cloud_siem/guide/google-cloud-config-guide-for-cloud-siem.md [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+
+**Warning**: Pub/Subs are subject to [Google Cloud quotas and limitations][10]. If the number of logs you have is higher than those limitations, Datadog recommends you split your logs over several topics. See [Monitor the Log Forwarding][11] for information on how to set up a monitor to notify when you are close to those limits.
```
Comment (cswatt) [inline content/en/security/cloud_siem/guide/google-cloud-config-guide-for-cloud-siem.md:81]:
```suggestion
**Warning**: Pub/subs are subject to [Google Cloud quotas and limitations][10]. If the number of logs you have is higher than those limitations, Datadog recommends you split your logs over several topics. See [Monitor the Log Forwarding][11] for information on how to set up a monitor to notify when you are close to those limits.
```

File: content/en/security/cloud_siem/guide/google-cloud-config-guide-for-cloud-siem.md [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+## Use Cloud SIEM to triage Security Signals
+
+Cloud SIEM applies out of the box detection rules to all processed logs, including the Google Cloud audit logs you have just set up. When a threat is detected with a Detection Rule, a Security Signal is generated and can be viewed in the Security Signals Explorer.
```
Comment (cswatt) [inline content/en/security/cloud_siem/guide/google-cloud-config-guide-for-cloud-siem.md:84]:
```suggestion
Cloud SIEM applies out-of-the-box detection rules to all processed logs, including the Google Cloud audit logs you have just set up. When a threat is detected with a detection rule, a Security Signal is generated and can be viewed in the Security Signals Explorer.
```

**Review by cswatt — CHANGES_REQUESTED** [review]
looks good, left small bits of feedback!

**Comment by maycmlee** [comment]
Thanks @cswatt! I applied all suggestions.

File: content/en/security/cloud_siem/guide/google-cloud-config-guide-for-cloud-siem.md [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+## Overview
+
+[Datadog Cloud SIEM][1] applies detection rules to all processed logs in Datadog to detect threats, like a targeted attack, a threat intel listed IP communicating with your systems, or an insecure resource modification. The threats are surfaced as Security Signals in the Security Signals Explorer for triaging.
+
+This guide walks you through the following steps so that you can start detecting threats with your Google Cloud audit logs:
+
+1. [Enable Data Access audit logs](#enable-data-access-audit-logs)
+1. [Create a new Google Cloud publish/subscription (pub/sub) system](#create-a-new-google-cloud-publishsubscription-pubsub-system)
+1. [Configure the pub/sub to send logs to Datadog](#configure-the-pubsub-to-send-logs-to-datadog)
+1. [Export logs from Google Cloud Logging to the pub/sub](#export-logs-from-google-cloud-logging-to-the-pubsub)
```
Comment (nimishaaa) [inline content/en/security/cloud_siem/guide/google-cloud-config-guide-for-cloud-siem.md:28]: should we add the last step here as "[Use Cloud SIEM to triage Security Signals]" similar to AWS guide?

File: content/en/security/cloud_siem/guide/google-cloud-config-guide-for-cloud-siem.md [hunk 6]
```diff
@@ -0,0 +1,109 @@
...
+## Use Cloud SIEM to triage Security Signals
+
+Cloud SIEM applies out-of-the-box detection rules to all processed logs, including the Google Cloud audit logs you have just set up. When a threat is detected with a detection rule, a Security Signal is generated and can be viewed in the Security Signals Explorer.
+
+- Go to the [Cloud SIEM Signals Explorer][12] to view and triage threats. See Security Signals Explorer for further details.
+- You can also use the [Google Cloud Audit Log dashboard][13] to investigate anomalous activity.
+- See [out-of-the-box detection rules][14] that are applied to your logs.
+- Create [new rules][15] to detect threats that match your specific use case.
+
+## Further reading
```
Comment (nimishaaa) [inline content/en/security/cloud_siem/guide/google-cloud-config-guide-for-cloud-siem.md:91]: should we link to the Google cloud docs here as well? https://docs.datadoghq.com/integrations/google_cloud_platform/#log-collection

**Review by nimishaaa — APPROVED** [review]

File: content/en/security/cloud_siem/guide/google-cloud-config-guide-for-cloud-siem.md (hunk 5 above)
Comment (maycmlee) [reply content/en/security/cloud_siem/guide/google-cloud-config-guide-for-cloud-siem.md:28]: Good catch, added it!

File: content/en/security/cloud_siem/guide/google-cloud-config-guide-for-cloud-siem.md (hunk 6 above)
Comment (maycmlee) [reply content/en/security/cloud_siem/guide/google-cloud-config-guide-for-cloud-siem.md:95]: Good idea, added it

**Review by cswatt — APPROVED** [review]

**Pull request merged**